webapp/packages/plugin-connection-template/src/locales/en.ts

```diff
@@ -8,5 +8,6 @@ export default [
   ['basicConnection_connectionDialog_passwordPlaceholder', 'password'],
   ['basicConnection_connectionDialog_connecting', 'Connecting...'],
   ['basicConnection_connectionDialog_connecting_message', 'Connecting to database...'],
+  ['basicConnection_connectionDialog_connect_success', 'Connection is established'],
```

Comment:
please use plugin prefix for naming `plugin_connection_template_connect_success`